lib/utils.js

```diff
@@ -375,5 +376,28 @@ const utils = (module.exports = {
     } else if (typeof process === 'object' && process.env) {
       return process.env[varName];
     }
+  },
+
+  createDeprecationWarning(message) {
+    let deprecationWarningDisplayed = false;
+
+    return () => {
+      if (deprecationWarningDisplayed) {
+        return;
+      }
+
+      deprecationWarningDisplayed = true;
```

Comment:
I don't think we should be cleaver about how much we show that warning.

Reply:
Then we can make a proper style for the warning in styles.js and make a util for printing them to the console.

Reply:
@sunesimonsen cool. I only did this based on what I saw `-sinon` do (https://github.com/unexpectedjs/unexpected-sinon/blob/c1bb48c459d6c8d501d8a63ca4c33a059aea6965/lib/unexpected-sinon.js#L1088) :)

Reply:
Let's keep it simple.